0006/README.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 0006
+title: Resiliency in Tinkerbell Provisioner.
+status: Ideation
+authors: Chitrabasu Khare <chitrabasukhare89@gmail.com>
+---
+
+## Summary:
+
+This RFD proposes to have a resilient framework for Tinkerbell Provisioner.
+
+
+## Goals:
+
+- Bring resiliency in Tinkerbell. 
+- Tinkerbell as a setup should be highly available.
+
+## Content: 
+
+- We would like to have a framework where Tinkerbell as a solution is fault-tolerant.
+- All most all of the core services are managed by provisioner. So, it will be good to have a framework where provisioner is highly available.
+- It can be a multinode provisioner solution with active-active or active-passive implementation.
```

Comment:
This is a good point, and maybe it can be a proposal by itself.. Even if I do not really understand it.

if you deploy postgress in RDS or it its own full tolerant way you can have 100 tink-server behind a proxy already today.

Reply:
Yes, this point demands a bit extensive implementation. but needs to be covered as a part of this proposal itself. 

IMO, we will have to create a number of subtask under this issue.